src/main/java/org/jenkinsci/test/acceptance/server/JenkinsControllerPoolProcess.java

```diff
@@ -57,8 +58,8 @@
     @Option(name="-n",usage="Number of instances to pool. >=1.")
     public int n = Integer.getInteger("count",1);
 
-    @Option(name="-socket",usage="Unix domain socket file to communicate with client")
-    public File socket = SOCKET;
+    @Option(name="-socket",usage="Unix domain socket file to communicate with client") @Inject @Named("socket")
```

Comment:
What do you suggest? I think we have two options:
a) remove the command line switch making the client/server interface the same
b) add another field that is injected by the command line and checked by a piece of code manually

I would prefer option a) as the command line switch was broken before (since no client counter part existed)